modules/module-mongodb/test/src/change_stream_utils.ts

```diff
@@ -85,7 +85,7 @@ export class ChangeStreamTestContext {
   }
 
   startStreaming() {
-    this.streamPromise = this.walStream.streamChanges();
+    return (this.streamPromise = this.walStream.streamChanges());
```

Comment:
This helps catch errors from the streaming process. 

An alternative would be to await the `dispose` method since that method awaits the `streamPromise`. Unfortunately the dispose also aborts the replication, which can make it tricky to detect errors.